context/app/static/js/components/Detail/ProvTableTile/ProvTableTile.jsx

```diff
@@ -12,6 +12,22 @@ function ProvTableTile(props) {
   // mapped fields are not included in ancestor object
   const entityData = useEntityData(uuid, elasticsearchEndpoint);
 
+  const types = entity_type === 'Donor' ? ['Sample', 'Dataset'] : ['Dataset'];
+
+  const descendantCounts = useMemo(() => {
```

Comment:
Should be able to use `descendant_counts` in the ES response instead of calculating it here.

Reply:
`descendant_counts` weren't showing / #793